easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2-GCC-6.3.0-2.27-hydra-opa.eb

```diff
@@ -0,0 +1,40 @@
+name = 'MVAPICH2'
+version = '2.2'
+versionsuffix = '-hydra-opa'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GCC', 'version': '6.3.0-2.27'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
+checksums = ['939b65ebe5b89a5bc822cdab0f31f96e']
+
+builddependencies = [('Bison', '3.0.4')]
+
+rdma_type = ''
+
+sanity_check_paths = {
+    'files': ["bin/mpiexec.hydra"],
+     'dirs' :[]
+}
+
+# Configure options
+configopts = [
+	"--sysconfdir=/etc \
```

Comment:
@henkela This looks quite site-specific...

Also, it shouldn't be a list (with just a single entry), but a string. We usually just list configure options on a single, or spread across multiple lines if it gets too long:

```python
configopts = "--sysconfdir=/etc ..."
configopts += "..."